common/include/pcl/common/tuple_utils.h

```diff
@@ -0,0 +1,108 @@
+/*
```

Comment:
This file is missing documentation for all public API as well as static_assert tests for the implementations

Reply:
Added doc. Where should I put these tests?